swap/swap_test.go

```diff
@@ -37,6 +38,11 @@ var (
 	loglevel = flag.Int("loglevel", 2, "verbosity of logs")
 )
 
+type Booking struct {
```

Comment:
I chose the name `Booking` since it was already being used by the test.

I'm not 100% sure this is the most descriptive name for this entity, though.

Reply:
`Booking` is fine to me, another option could be `entry`.

In any case, if there is no reason to have this name exported (upper case first letter), and I can't currently see no reason to have this exported (only used within this test), it should be lower case.

Please also add a comment about this struct (what is it). Especially if it *would* be exported, all exported objects should be commented.

Reply:
You're right, there's no reason to export this. I'll change it, but I will still add comments for clarity.

Reply:
Done.